docs/content/kubernetes-tutorial.md

```diff
@@ -10,6 +10,9 @@ For the purpose of the tutorial we will deploy two policies that ensure:
 
 - Ingress hostnames must be whitelisted on the Namespace containing the Ingress.
 - Two ingresses in different namespaces must not have the same hostname.
+  - Note: this behavior may suffer from a race condition in which two ingress with the same
+    hostname may both be allowed into the cluster.  This is because OPA evaluates policies
+    against an eventually-consistent view of the global cluster state.
```

Comment:
I'd like to tweak the wording on this because this is an architectural issue with Kubernetes itself--every single webhook admission controller has the same limitation. There is no consistency across Kubernetes resources, it's eventually consistent by design.

Something like:

> Kubernetes does not guarantee consistency across resources. If two ingresses are created in parallel, there is no guarantee that OPA (or any other admission controller) will observe the creation of one ingress before the other. This means that it's not possible to enforce these policies during admission control 100% of the time. There will be a small window of time (usually on the order of milliseconds) when the eventually consistent cache inside of OPA (or any other admission controller) is out-of-date. To catch these violations we recommend you periodically audit the state of the cluster against your clusters. Offline auditing is one of the features provided by the [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) project.

Reply:
Thanks for looking at this.

I don't think I know enough to make statements about the capabilities of all other admission controllers, or about the length of time where the race condition might occur.

If you'd like to add that language in a commit, please leave my name off of it.   Thanks.

Reply:
No problem! I'll update the docs myself as I'm (perhaps too) familiar with admission control in Kubernetes. Thanks for bringing this up.